packages/web-sdk/src/transports/otlp/transform/semanticResourceAttributes.ts

```diff
@@ -0,0 +1,39 @@
+import { SemanticAttributes, SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+
+/**
+ * Seems currently to be missing in the semantic-conventions npm package.
+ * See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#todos
+ *
+ * Took the few attributes as defined in the docs
+ */
+export const SemanticBrowserAttributes = {
+  BROWSER_BRANDS: 'browser.brands', // TODO: Q: shall we add this to meta.ts => navigator.userAgentData.brands. !The spec is still experimental!
+  BROWSER_PLATFORM: 'browser.platform',
+  BROWSER_MOBILE: 'browser.mobile',
+  BROWSER_USER_AGENT: 'browser.user_agent', // TODO: Q: shall we add this to meta.ts => parser.getUA()
+  BROWSER_LANGUAGE: 'browser.language', // TODO: Q: shall we add this to meta.ts => window.navigator.language
+} as const;
+
+export const faroResourceAttributes = {
+  BROWSER_MOBILE: SemanticBrowserAttributes.BROWSER_MOBILE, // browser.mobile
+  BROWSER_NAME: 'browser.name', // FARO
+  BROWSER_VERSION: 'browser.version', // FARO
+  BROWSER_PLATFORM: SemanticBrowserAttributes.BROWSER_PLATFORM, // 'browser.platform'
+
+  TELEMETRY_SDK_NAME: SemanticResourceAttributes.TELEMETRY_SDK_NAME, // 'telemetry.sdk.name',
```

Comment:
Note:
Unfortunately `SemanticResourceAttributes` and `SemanticAttributes` are all of type string. 
The object which contains the real values is auto generated. 

This means using widens the derive type to string instead of a union of the actual values. 
So maybe we should use the respective string values instead of retrieving the form the npm module.